modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/list-item-with-icon/index.js

```diff
@@ -0,0 +1,22 @@
+const plusIcon = fragmentElement.querySelector('#plus-icon');
+const minusIcon = fragmentElement.querySelector('#minus-icon');
+const customIcons = fragmentElement.getElementsByClassName('custom-icon');
+const collapseDiv = fragmentElement.querySelector('#collapseDiv');
```

Comment:
```suggestion
const faqList = fragmentElement.querySelector('#faq-list');
```

Use the suggestion from above.

Reply:
One advice for this, follow the patterns of name.

Ex: You used #plus-icon and #minus-icon as hiphen-case

But for this one, you used camelCase. 

In both cases, we need to pick one and follow

Reply:
I didn't pay attention to it, thank you for reminding me.